azure-pipelines.yml

```
@@ -55,7 +63,7 @@ stages:
 
     - script: |
```

Windows agent failed at this point. Using powershell instead of cmd will allow the same script to run on Windows as works on bash.

```suggestion
    - pwsh: |
```